src/main/webapp/app/tasks/local-task.service.ts

```diff
@@ -16,25 +16,43 @@ export class LocalTaskService implements TaskService {
 
   create(name: string): Observable<Task> {
     const tasks = this.readTasks();
-    const task = {id: uuid(), name};
+    // Added new key value pair
+    const task = {id: uuid(), name, isCompleted: false};
     tasks.push(task);
     this.writeTasks(tasks);
     return of(task);
   }
 
+  update(id: string, task: Task): Observable<Task> {
```

Comment:
Why did you decide to do the change by mutating the task?

Reply:
I am updating the task by the status of isCompleted inside the array. That's why I have used this method.